src/test/java/seedu/address/logic/parser/CommandParserTestUtil.java

```diff
@@ -19,8 +19,8 @@ public static void assertParseSuccess(Parser<? extends Command> parser, String u
         try {
             Command command = parser.parse(userInput);
             assertEquals(expectedCommand, command);
-        } catch (ParseException pe) {
-            throw new IllegalArgumentException("Invalid userInput.", pe);
+        } catch (ParseException e) {
+            throw new IllegalArgumentException("Invalid userInput.", e);
```

Comment:
Why did you changed `pe` to `e` here, but not for the method below?

Reply:
Previously I adjust the catch clause to also handle CommandExcption, thus the variable for exception should be more general. But later I realized that Parser classes should only throw ParseException.

Reply:
There's no difference here.